Resources/Prototypes/Entities/Mobs/Species/diona.yml

```diff
@@ -0,0 +1,54 @@
+- type: entity
+  save: false
+  name: Urist McPlants
+  parent: BaseMobOrganic
+  id: BaseMobDiona
+  abstract: true
+  components:
+  - type: Humanoid
+    species: Diona
+  - type: Hunger
+  - type: Thirst
+    baseDecayRate: 0.3
+  - type: Icon
+    sprite: Mobs/Species/Diona/parts.rsi
+    state: full
+  - type: Body
+    template: HumanoidTemplate
+    preset: DionaPreset
+  - type: Damageable
+    damageContainer: Biological
+    damageModifierSet: Diona
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+      - id: FoodMeatPlant
+        amount: 5
+  - type: Bloodstream
+    bloodReagent: Water
+    bloodlossDamage:
+      types:
+        Bloodloss:
+          1
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -0.25
+  - type: Inventory
+    templateId: diona
+  - type: MovementSpeedModifier
```

Comment:
Are they the same mass (70 kg) as other species? If not you can adjust their physics component